### ISSUE TITLE
Add a template generic example script as a quick start with Ignite

### ISSUE THREAD

**Issue by vfdev-5** [pytorch/ignite#1007]
## 🚀 Feature

Idea is to simplify user's learning curve and provide a template example that can be copied by the user and modified to his/her needs. Something like that but an executable working example:
```

def training():

    train_loader, val_loader = get_dataflow(fold_index=fold_index, **kwargs)
    model, optimizer, criterion, lr_scheduler = initialize_model(**kwargs)
    metrics = {
         "accuracy": Accuracy(),
    }

    trainer = create_supervised_trainer(model, optimizer, criterion)

    @trainer.on(Events.EPOCH_COMPLETED)
    def update_lr_scheduler(_):
        lr_scheduler.step()

    evaluator = create_supervised_evaluator(model, metrics=metrics)

    @trainer.on(Events.ITERATION_COMPLETED(every=validate_every))
    def validate(trainer):
        evaluator.run(val_loader)
        metrics = evaluator.state.metrics
        print("After {} iterations, binary accuracy = {:.2f}"
              .format(trainer.state.iteration, metrics['accuracy']))

    trainer.run(train_loader, max_epochs=kwargs.get(max_epochs, 100))
    

```

**Comment by sdesrozis** [comment]
I agree!! That is an excellent idea! We have so much options and possibilities to tune. We can’t provide only helpers.

**Comment by louis-she** [comment]
I'm thinking of a cli command tool may be helpful for this. Like:
```
ignite project new --template mnist path_of_folder
```
or we can use a interactive way like `cookiecutter` does.

**Comment by vfdev-5** [comment]
@louis-she today we have also [pytorch-ignite code generator](https://code-generator.pytorch-ignite.ai/) that should be a sort of interactive cookiecutter. Please take a look and if you have any feedback on how to improve it, we would be happy to have it. Thanks

**Comment by bibhabasumohapatra** [comment]
> @louis-she today we have also [pytorch-ignite code generator](https://code-generator.pytorch-ignite.ai/) that should be a sort of interactive cookiecutter. Please take a look and if you have any feedback on how to improve it, we would be happy to have it. Thanks

[pytorch-ignite code generator](https://code-generator.pytorch-ignite.ai/)  looks great!!

**Comment by louis-she** [comment]
> @louis-she today we have also [pytorch-ignite code generator](https://code-generator.pytorch-ignite.ai/) that should be a sort of interactive cookiecutter. Please take a look and if you have any feedback on how to improve it, we would be happy to have it. Thanks

Wow the generator site is way more cooler thant cookiecutter 💯
